paver.conf

```diff
@@ -1,4 +1,4 @@
-BASE_REQUIREMENTS='chevah-brink==0.63.3 paver==1.2.4'
-PYTHON_CONFIGURATION='default@2.7.13.4f815b5:solaris10@2.7.8.4f815b5'
+PYTHON_CONFIGURATION='default@2.7.13.25cf4cf:solaris11@2.7.13.c3d8a7d:solaris10@2.7.8.c3d8a7d:freebsd11@2.7.13.641bbdca'
+BASE_REQUIREMENTS='chevah-brink==0.63.4 paver==1.2.4'
```

Comment:
For disk space reasons, I would go with specific versions for specific changes and then merging them all together when there are changes that impact all platforms that require rebuilding all packages.

However, when publishing new generic versions, I would only do it after merging with  master and getting a git rev for the squashed commit. So that `git show` for that specific revision looks as informative as possible. It's a bit of a chicken and egg problem for python-package, but shouldn't be an issue for the other repos.